api/src/test/resources/org/openmrs/module/ohricore/include/OHRI-StandardDataset-original.xml

```diff
@@ -0,0 +1,442 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
@smallgod What is this file for leverage version control to track changes not file names

Reply:
Oh yes, I have removed this. Thanks